.github/workflows/check_in_artifact.yml

```diff
@@ -117,7 +117,7 @@ jobs:
         run: |
           git checkout stash -- .
           git add ${{ inputs.artifact_save_path }}
-          git commit -m "Check in artifacts$COMMIT_MESSAGE_DESCRIPTION"
+          git commit -m --allow-empty "Check in artifacts$COMMIT_MESSAGE_DESCRIPTION"
```

Comment:
When comparing the updated artifacts with `master`, there are changes, so we go to the branch which gets used for opening the PR. However, if the said changes have been committed to this branch by a previous run of the same action (say, in a different PR), then this `git commit` will fail because there are no changes to commit. So I'm adding this `--allow-empty` so that the commit step doesn't fail.

This is a patch more than anything else. I think for our purposes, this is fine, because we don't care about how many commits are pushed to the branch, just that the changes reflect what we expect.